adam-core/src/main/scala/org/bdgenomics/adam/models/Interval.scala

```diff
@@ -41,4 +42,19 @@ trait Interval {
    */
   def width: Long = end - start
 
+  /**
+   * Need to implement getStart function from Locatable. (1-based start position. closed)
```

Comment:
No, this allows us to supply `ReferenceRegion`s to the Hadoop-BAM indexed input format code without going through an intermediate class.